botoffliner.lua

```diff
@@ -336,20 +336,20 @@ function ExecuteCommand( tUser, sCmd, sData )
 			Core.SendPmToUser( tUser, tCfg.sBotName, "Sorry! The user "..tBreak[1].." is not a moderator." )
```

Comment:
Also, you will have to shift this check for moderator into the loop, because every nick `tBreak` should be a moderator.